tests/debian/test.yaml

```diff
@@ -9,6 +9,12 @@ actions:
     suite: {{ $suite }}
     variant: minbase
     merged-usr: true
+{{- if eq $tool "mmdebstrap" }}
+    dpkg-opts:
+      - path-exclude=/usr/share/man/*
+    apt-opts:
+      - Apt::Install-Recommends "true"
+{{- end }}
```

Comment:
Ideally for a more complete test you should verify if these options were really applied/ended up in the expected config files

Reply:
@iwamatsu I'll merge this now so we don't block a potential next release; but would be great if you could do a PR later on with a few more tests for this functionality